web-previewer/package.json

```diff
@@ -12,39 +12,21 @@
         "@tailwindcss/forms": "^0.3.3",
         "@tailwindcss/postcss7-compat": "^2.2.17",
         "core-js": "^3.6.5",
+        "ethers": "^5.5.1",
         "lodash-es": "^4.17.21",
         "tailwindcss": "npm:@tailwindcss/postcss7-compat",
         "vue": "^3.2.6",
-        "vue-router": "4",
-        "web3": "^1.6.0"
+        "vue-router": "4"
     },
     "devDependencies": {
         "@vue/cli-plugin-babel": "~4.5.0",
-        "@vue/cli-plugin-eslint": "~4.5.0",
         "@vue/cli-service": "~4.5.0",
         "@vue/compiler-sfc": "^3.0.0",
         "autoprefixer": "^9",
         "babel-eslint": "^10.1.0",
-        "eslint": "^6.7.2",
-        "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-vue": "^7.0.0",
         "postcss": "^7",
         "prettier": "2.4.1"
     },
-    "eslintConfig": {
-        "root": true,
-        "env": {
-            "node": true
-        },
-        "extends": [
-            "plugin:vue/vue3-essential",
-            "eslint:recommended"
-        ],
-        "parserOptions": {
-            "parser": "babel-eslint"
-        },
-        "rules": {}
-    },
```

Comment:
Any particular reason why this + `.eslintrc.js` was removed? 

Would prefer to keep that config and run `yarn lint` + `yarn prettier` to make it match the rest of the codebase :-)

Reply:
I removed it because it throws linting errors in ClaimControlls, PreviewControlls and ConnectWallet. I tried to fix these errors but then the code was no longer running.
So for now I disabled es-lint for the corresponding lines. Maybe you can check how to fix this :)